src/test/java/com/neuvector/ScannerTest.java

```diff
@@ -45,14 +57,30 @@ public void scanRegistryTest(){
         String repositoryTag = "3.6";
         String nvScannerImage = "neuvector/scanner:latest";
         String nvRegistryURL = registryURL;
-        String nvRegistryUser = "xxx";
-        String nvRegistryPassword = "xxx";
+        String nvRegistryUser = "";
+        String nvRegistryPassword = "";
         String mountPath = "/temp";  //mountPath is an optional parameter. It will use "/var/neuvector" by default.
 
         Registry registry = new Registry(registryURL, regUser, regPassword,repository,repositoryTag);
         NVScanner scanner = new NVScanner(nvScannerImage, nvRegistryURL, nvRegistryUser, nvRegistryPassword, mountPath);
 
         ScanRepoReportData scanReportData = Scanner.scanRegistry(registry, scanner, license);
+
+        File file = new File(mountPath + "/scan_result.json");
+        UserPrincipal user = getUserPrincipal(mountPath, file);
+
+        assertTrue(user.getName().equals("root") || user.getName().contains(System.getProperty("user.name")));
         assertTrue( scanReportData != null );
     }
+
+    private static UserPrincipal getUserPrincipal(String mountPath, File file) throws IOException {
+        UserPrincipal user = null;
+        if (file.exists()) {
+            Path path = Paths.get(mountPath + "/scan_result.json");
```

Comment:
We should either have `scan_result.json` hardcoded like here, or use the second argument to the method. This method is checking if the passed in file exists, but than gets attributes from a hardcoded file..

Reply:
I removed the file attribute from here: a6e52d49e0973fb081e8cbe3ee4cc56a3287e22b